### ISSUE TITLE
Server-side velocity setter

### ISSUE THREAD

**Issue by zziger** [altmp/altv-issues#1362]
### Description of the problem

Sometimes it's useful to change velocity of some entity, for example to immediately stop it. It would be nice to have that ability as a velocity setter (because we already have velocity getter)

### Desired solution for the problem

.velocity setter

### Alternatives you considered

Setting the velocity via native by sending an event to the netowner

**Comment by FabianTerhorst** [comment]
Internally we could also just send a event to the netowner. We cant change the velocity by modifying the sync data because the netowner would override it.

**Comment by zziger** [comment]
> Internally we could also just send a event to the netowner. We cant change the velocity by modifying the sync data because the netowner would override it.

I guess even with this implementation that would be more effective (cuz no natives and custom events involved), but what about situations where the vehicle is out of anyone's stream range. Does it preserve it's velocity until it gets streamed by someone?

**Comment by FabianTerhorst** [comment]
Yes it theoretically would keep the velocity when someone comes in range.
